community/xcb-util-xrm/build

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr \
+    --sysconfdir=/etc \
+    --mandir=/usr/share/man \
```

Comment:
Are any man pages installed? If not, this can be removed.

Also check to see what the default value of this is. It could very well use `--prefix`.